src/Vendr.PaymentProviders.Worldpay/Vendr.PaymentProviders.Worldpay.csproj

```diff
@@ -73,14 +76,16 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
-    <Reference Include="Vendr.Core, Version=0.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Vendr.Core.0.1.0-alpha0336\lib\net472\Vendr.Core.dll</HintPath>
+    <Reference Include="Vendr.Core, Version=1.8.1.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Vendr.Core.1.8.1\lib\net472\Vendr.Core.dll</HintPath>
     </Reference>
-    <Reference Include="Vendr.Core.Web, Version=0.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Vendr.Core.Web.0.1.0-alpha0336\lib\net472\Vendr.Core.Web.dll</HintPath>
+    <Reference Include="Vendr.Core.Web, Version=1.8.1.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Vendr.Core.Web.1.8.1\lib\net472\Vendr.Core.Web.dll</HintPath>
```

Comment:
I need to check what base version is best for this as you generally want to have a dependency on the lowest Vendr version you can. I think the last time payment providers changed was in 1.4.0 but I'll need to double check that.

Reply:
Yup, looks like 1.4.0 is the preferred minimum atm.

Reply:
Cool - will update this